dist/index.html

```diff
@@ -61,14 +61,13 @@
         var key = encodeURIComponent($('#input_apiKey')[0].value);
         log("key: " + key);
         if(key && key.trim() != "") {
+            var apiKeyAuth = new SwaggerClient.ApiKeyAuthorization("api_key", key, "query");
+            window.swaggerUi.api.clientAuthorizations.add("api_key", apiKeyAuth);

```

Comment:
`window.swaggerUi` is that instance. What I did is, in SwaggerUi class itself there is no reference to it anymore (because the class should not depend on an instance)